### PR TITLE
Application: use startup

### PR DESCRIPTION
**Pull request by danirabbit**

### CODE REVIEW

File: src/Application.vala [hunk 1]
```diff
@@ -53,6 +53,21 @@ public class Tasks.Application : Gtk.Application {
...
         add_main_option_entries (OPTIONS);
+    }
+
+    protected override void startup () {
+        base.startup ();
+
+        Hdy.init ();
+
+        var granite_settings = Granite.Settings.get_default ();
+        var gtk_settings = Gtk.Settings.get_default ();
```
Comment (Marukesu) [inline src/Application.vala:64]:
```suggestion
        unowned var granite_settings = Granite.Settings.get_default ();
        unowned var gtk_settings = Gtk.Settings.get_default ();
```

File: src/Application.vala [hunk 2]
```diff
@@ -53,6 +53,21 @@ public class Tasks.Application : Gtk.Application {
...
+
+        gtk_settings.gtk_application_prefer_dark_theme = granite_settings.prefers_color_scheme == Granite.Settings.ColorScheme.DARK;
+
+        granite_settings.notify["prefers-color-scheme"].connect (() => {
+            gtk_settings.gtk_application_prefer_dark_theme = granite_settings.prefers_color_scheme == Granite.Settings.ColorScheme.DARK;
+        });
```
Comment (Marukesu) [inline src/Application.vala:70]: i think this is a good place to use the reduced enum name syntax.

```suggestion
        gtk_settings.gtk_application_prefer_dark_theme = granite_settings.prefers_color_scheme == DARK;

        granite_settings.notify["prefers-color-scheme"].connect ((obj) => {
            gtk_settings.gtk_application_prefer_dark_theme = ((Granite.Settings) obj).prefers_color_scheme == DARK;
        });
```

Comment (danirabbit) [reply src/Application.vala:70]: Ah yeah that's much nicer. Good call!

**Review by Marukesu — APPROVED** [review]
LGTM.

**Pull request merged**